src/mediapipe_internal/mediapipegraphexecutor.cpp

```diff
@@ -689,20 +693,32 @@ static Status deserializeTensor(const std::string& requestedName, const KFSReque
                 SPDLOG_DEBUG("[servable name: {} version: {}] {}", request.model_name(), request.model_version(), details);
                 return Status(StatusCode::INVALID_PRECISION, details);
             }
-            size_t expectedBytes = 1;
-            bool expectedBufferSizeValid = computeExpectedBufferSizeReturnFalseIfOverflow<py::ssize_t>(shape, precision.size(), expectedBytes);
-            if (!expectedBufferSizeValid) {
-                const std::string details = "Provided shape and datatype declare too large buffer.";
-                SPDLOG_DEBUG("[servable name: {} version: {}] {}", request.model_name(), request.model_version(), details);
-                return Status(StatusCode::INVALID_CONTENT_SIZE, details);
+            size_t expectedBytes;
+            if (precision == ov::element::Type_t::string) {
+                expectedBytes = 0;
+                for (auto contents : request.inputs(inputIndex).contents().bytes_contents()) {
+                    expectedBytes = expectedBytes + contents.size() + sizeof(uint32_t);
```

Comment:
Isn't size written in first bytes of the input? If so I suggest keeping the order also here.
```suggestion
                    expectedBytes += sizeof(uint32_t) + contents.size();
```